apps/frontend/src/app/5_pages/LandingPage/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+
+import { t } from 'i18next';
+import Carousel from 'react-multi-carousel';
+import 'react-multi-carousel/lib/styles.css';
+
+import { Button, ButtonSize, ButtonStyle } from '@sovryn/ui';
+
+import { WIKI_LINKS } from '../../../../../constants/links';
+import { translations } from '../../../../../locales/i18n';
+import styles from './Banner.module.css';
+import { LandingPromoCard } from './components/LandingPromoCard/LandingPromoCard';
+
+export const Banner: FC = () => (
+  <div className="w-full relative pb-7">
+    <Carousel
+      arrows={false}
+      draggable
+      partialVisible
+      focusOnSelect={false}
+      responsive={{
+        large: {
+          breakpoint: { max: 4800, min: 0 },
+          items: 1,
+          slidesToSlide: 1,
+        },
+      }}
+      minimumTouchDrag={80}
+      swipeable
+      className="static"
+      renderDotsOutside
+      showDots
+      autoPlay={true}
+      dotListClass={styles.dot}
+      autoPlaySpeed={15000}
+      infinite
+    >
+      <LandingPromoCard
+        heading={t(translations.landingPage.promotions.sov.title)}
+        description={t(translations.landingPage.promotions.sov.description)}
+        actions={
+          <>
+            <Button
+              style={ButtonStyle.secondary}
+              size={ButtonSize.large}
+              text={t(translations.landingPage.promotions.sov.cta)}
+              href="/convert?from=rbtc&to=sov"
+            />
```

Comment:
Change this button to use react router - currently it works as a link refreshing entire dapp when moving to another page